tests/test_pyglmnet.py

```diff
@@ -95,9 +95,9 @@ def test_multinomial():
                  tol=1e-10)
     X = np.array([[-1, -2, -3], [4, 5, 6]])
     y = np.array([1, 0])
+
```

Comment:
it's usually not considered good practice to touch parts of code not related to the PR. Maybe I'm being a bit too nitpicky ;)